sample-libs/Samples.DatabaseHelper/DbCommandInterfaceExecutor.cs

```diff
@@ -0,0 +1,45 @@
+using System.Data;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Samples.DatabaseHelper
+{
+    public class DbCommandInterfaceExecutor : DbCommandExecutor<IDbCommand>
+    {
+        private static readonly Task CompletedTask = Task.FromResult(0);
+
+        public override string CommandTypeName => nameof(IDbCommand);
+
+        public override bool SupportsAsyncMethods => false;
+
+        public override void ExecuteNonQuery(IDbCommand command) => command.ExecuteNonQuery();
+
+        public override Task ExecuteNonQueryAsync(IDbCommand command) => CompletedTask;
+
+        public override Task ExecuteNonQueryAsync(IDbCommand command, CancellationToken cancellationToken) => CompletedTask;
```

Comment:
nit: You should be able to use the `Task.CompletedTask` property here instead of creating your own

Reply:
`Task.CompletedTask` was added in .NET Framework 4.6, so we can't use it when targeting 4.5.x.

Reply:
Ah, good to know. Thanks for the explanation!